packages/widget/src/components/Step/StepTimer.tsx

```diff
@@ -1,7 +1,8 @@
 import type { LiFiStepExtended } from '@lifi/sdk';
 import { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
-import { useTimer } from 'react-timer-hook';
+import pkg from 'react-timer-hook';
```

Comment:
added this as there was an error from one of the examples about this package being common js